UPDATING.md

```diff
@@ -3,6 +3,12 @@
 This file documents any backwards-incompatible changes in Superset and
 assists people when migrating to a new version.
 
+## Superset 0.32.0
```

Comment:
@mistercrunch this PR got into 0.31 so this should be in the 0.31 section. Also I think we should call out the specific dependencies that were removed, we were impacted by removing `flower` would be good to list that out.

Reply:
Oh I assumed this would hit 0.32... let me change that then...

Reply:
Here: https://github.com/apache/incubator-superset/pull/7117